pbsports/Azure-AD-B2C-page-templates
# Mvp 2314 enable auto password recovery

## Purpose

* enable self-password recovery for the user (for now only TRIQ Dev can be tested)

## Does this introduce a breaking change?

```
[x] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[x] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
* Delete and re-install TRIQ Dev
* Try to reset password from an existing account
* User should be logged in immediately after password change

* Test the code
No need to test the code since it's mostly html/css baby stuff

## What to Check
Verify that the following are valid
* Create new user, sign in, sign out => try to reset password via "forgot password?" link
* New password must be working for login, old password may not be used anymore
* Design of user flow (password reset) must be similar to sign in / sign up forms
